### ISSUE TITLE
[Improvement] Ability to override url for the "view" link

### ISSUE THREAD

**Issue by s-g-k** [sanity-io/sanity-plugin-dashboard-widget-netlify#8]
For sites with custom domains, it would be great to be able to override the `url` prop with a custom domain.

Something along these lines:

```
{
  name: 'netlify',
  options: {
    title: 'Deployments',
    sites: [
      {
        title: 'Website',
        apiId: 'xxxx-xxxxx',
        buildHookId: 'xxxxxxxxx',
        name: 'website',
        url: 'https://example.com/'
      }
    ]
  }
}
```

**Comment by timurrustamov** [comment]
Featured in https://github.com/sanity-io/sanity-plugin-dashboard-widget-netlify/pull/9 ;)

**Comment by rexxars** [comment]
Released in [v1.1.0](https://github.com/sanity-io/sanity-plugin-dashboard-widget-netlify/releases/tag/v1.1.0) thanks to @timurrustamov 👍
